mii/server.py

```diff
@@ -318,6 +318,17 @@ def _initialize_service(self,
                                            mii_configs,
                                            lb_config))
 
+            if mii_configs.restful_api_port > 0:
```

Comment:
What if we used a `mii_configs.enable_restful_api` here instead of checking if the port is greater than 0? (and we could set a default port number for the restful api)

Reply:
I will change as proposed (See also the above comment)